integration/v3_grpc_test.go

```diff
@@ -147,6 +148,55 @@ func TestV3CompactCurrentRev(t *testing.T) {
 	}
 }
 
+// TestV3HashKV ensures that multiple calls of HashKV on same node return same hash and compact rev.
+func TestV3HashKV(t *testing.T) {
+	defer testutil.AfterTest(t)
+	clus := NewClusterV3(t, &ClusterConfig{Size: 1})
+	defer clus.Terminate(t)
+
+	kvc := toGRPC(clus.RandClient()).KV
+	mvc := toGRPC(clus.RandClient()).Maintenance
+
+	var rev int64
+	for i := 0; i < 10; i++ {
+		resp, err := kvc.Put(context.Background(), &pb.PutRequest{Key: []byte("foo"), Value: []byte("bar" + strconv.Itoa(i))})
+		if err != nil {
+			t.Fatal(err)
+		}
+		rev = resp.Header.Revision
+	}
+
+	// ensure appliedIdx is sync with committedIdx
+	_, err := kvc.Range(context.Background(), &pb.RangeRequest{Key: []byte("foo")})
```

Comment:
this isn't necessary for a single member cluster; there won't be an ack for the put until it's synced